draft-ietf-ipsecme-ikev2-multiple-ke-00.xml

```diff
@@ -825,17 +827,17 @@ HDR(IKE_FOLLOWUP_KE), SK {KEi(3),
                 <xref target="RFC7296" /> describe how IKEv2 handles this situation.  In a nutshell
                 IKEv2 follows the rule that if in case of simultaneous rekeying two identical new
                 IKE SAs (or two pairs of Child SAs) are created, then one of them should be deleted. 
-                Which one is to be deleted is determined by comparing the values of four nonces, 
-                that were used in the colliding CREATE_CHILD_SA exchanges - the IKE SA (or pair of Child SAs) 
-                that was created by the exchange in which the smallest nonce was used should be deleted by 
+                Which one is to be deleted is determined by comparing the values of four nonces
+                that are used in the colliding CREATE_CHILD_SA exchanges. The IKE SA (or pair of Child SAs) 
+                that is created by the exchange in which the smallest nonce is used should be deleted by 
                 the initiator of this exchange.
                 </t>
 
-                <t> With multiple key exchanges the SAs are not yet created when the CREATE_CHILD_SA is completed,
+                <t> With multiple key exchanges, the SAs are not yet created when the CREATE_CHILD_SA is completed,
                 they would be created only after the series of IKE_FOLLOWUP_KE exchanges is finished.
-                For this reason if additional key exchanges were negotiated in the CREATE_CHILD_SA initiated by the losing side, 
+                For this reason, if additional key exchanges are negotiated in the CREATE_CHILD_SA initiated by the side having the smallest nonce, 
                 there is nothing to delete and this side just stops the rekeying process - this side MUST NOT initiate 
-                IKE_FOLLOWUP_KE exchange with next key exchange.
+                IKE_FOLLOWUP_KE exchange in the next key exchange.
```

Comment:
Ok, how about this?
```
For this reason, if additional key exchanges are negotiated in the CREATE_CHILD_SA that is initiated
by the side having the smallest nonce, there is nothing to delete and this side just stops the rekeying
process and it MUST NOT initiate the IKE_FOLLOWUP_KE exchange.
```

Reply:
I'd refine a bit:

For this reason, if additional key exchanges are negotiated in the CREATE_CHILD_SA exchange in which the smallest nonce was used, then, since there is nothing to delete yet, the initiator of this exchange just stops the rekeying process and it MUST NOT initiate the IKE_FOLLOWUP_KE exchange.

Reply:
Thanks.